### ISSUE TITLE
An update of ThreeJS to 112 breaks the VTO example

### ISSUE THREAD

**Issue by ThorstenBux** [jeeliz/jeelizFaceFilter#154]
If you are submitting a bug report, please provide:

* A clear and concise description of what the bug is: what you expect and what happens,
Updating ThreeJS to version 112 breaks the shader code provided here: https://github.com/jeeliz/jeelizFaceFilter/blob/master/demos/threejs/VTO/JeelizThreeGlassesCreator.js#L39
The browser log shows
```
index.js:1472 THREE.WebGLProgram: shader error:  0 35715 false gl.getProgramInfoLog invalid shaders  THREE.WebGLShader: gl.getShaderInfoLog() fragment
ERROR: 0:1214: 'GL_OES_standard_derivatives' : extension is disabled
ERROR: 0:1214: 'GL_OES_standard_derivatives' : extension is disabled
1: #extension GL_EXT_shader_texture_lod : enable
```
I tried to work around that but couldn't find a suitable fix. I thought I'll let you know maybe you would like to fix it.
* How to reproduce it (a code chunk or a link would be great),
Update example to ThreeJS 112

**Comment by xavierjs** [comment]
Hi,

I have pushed a fix:
- I use the stable version of THREE.js r112 (not the one on Github, but the one to download on https://threejs.org/ )
- I have updated the VTO demo on the github repo to use THREE.js r112
- It seems that some feature with THREE.MeshBasicMaterial and THREE.MeshLambertMaterial are broken. In our case we want to use them without lighting and envmap enabled. So I have replace the MeshBasicMaterial by a MeshStandardMaterial. I think the current THREE.js trend is to use this material model for everything which requires a realistic 3D rendering.
